content/250-postgres/250-overview.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: 'Overview'
+metaTitle: 'Overview | Prisma Postgres'
+metaDescription: 'Learn everything you need to know about Prisma Postgres.'
+tocDepth: 3
+toc: true
+---
+
+
+[Prisma Postgres®](https://www.prisma.io/postgres?utm_source=docs) is a managed PostgreSQL database service that easily lets you create a new database, interact with it through Prisma ORM, and build applications that start small and cheap but can scale to millions of users.
+
+It supports the following workflows:
+
+- Schema migrations and queries (via [Prisma ORM](https://www.prisma.io/orm))
+- Connection pooling and caching (via [Prisma Accelerate](https://www.prisma.io/accelerate))
+- Real-time database change events (via [Prisma Pulse](https://www.prisma.io/pulse))
+
+## Pricing 
+
+To learn about how pricing works with Prisma Postgres, visit the [pricing page](https://www.prisma.io/pricing) and [billing documentation](/platform/about#billing).
```

Comment:
@nikolasburk, we discussed on Tuesday to edit the Platform docs as needed to include all the pricing and billing details we want to share, similar to https://supabase.com/docs/guides/platform/org-based-billing.

I see that we didn't really make a lot of changes to that page, was everything already on it? Also, would it make sense to rename that from About to e.g. General, or similar? I find "About" a bit unintuitive.

Reply:
Yes, this is actually[ still on my list](https://linear.app/prisma-company/issue/DA-2525/update-billing-in-platformabout), haven't gotten to that yet. 

> I see that we didn't really make a lot of changes to that page, was everything already on it? Also, would it make sense to rename that from About to e.g. General, or similar? I find "About" a bit unintuitive.

Will think about this as well! I'm currently doing final edits/polish of the docs so this kind of feedback is still very helpful 🙏